packages/manager/apps/nasha-react/src/pages/details/[serviceName]/DatePretty.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { useShell } from '@ovh-ux/manager-react-core-application';
+
+export type DatePrettyProps = {
+  date: Date;
+  formatOptions?: Intl.DateTimeFormatOptions;
+};
+
+export default function DatePretty({
+  date,
+  formatOptions,
+}: DatePrettyProps): JSX.Element {
+  const [dateLabel, setDateLabel] = useState('');
```

Comment:
Maybe we could use the localeDateString with undefined as the first value for the initial value

```
useState(date.getLocaleDateString(undefined, formatOptions));
```